.github/CODEOWNERS

```diff
@@ -0,0 +1,62 @@
+# Order is important; the last matching pattern takes the most precedence.
+
+.github            @alpenlabs/admins             # only admins should own the CODEOWNERS
+.gitignore         @alpenlabs/admins
+**/Cargo.toml      @alpenlabs/dependencies       # dependencies and metadata
+**/Cargo.lock      @alpenlabs/dependencies       # updates
+.config/           @alpenlabs/rust @alpenlabs/ci # config for rustdev and CI
+**/*.toml          @alpenlabs/rust               # config files
+
+# CI/Local ENV
+.github/workflows/     @alpenlabs/ci
+.github/actions/       @alpenlabs/ci
+.github/dependabot.yml @alpenlabs/ci
+Makefile               @alpenlabs/rust @alpenlabs/ci @alpenlabs/bin
+codecov.yml            @alpenlabs/ci
+contrib/               @alpenlabs/ci @alpenlabs/bin
+tests/                 @alpenlabs/rust @alpenlabs/ci
+
+# Docker/Containers
+**/Docker*         @alpenlabs/docker
+docker/            @alpenlabs/docker
+
+# Libs
+crates/bridge*     @alpenlabs/bridge
+crates/btc*        @alpenlabs/btc
+crates/common      @alpenlabs/rust
+crates/chaintsn    @alpenlabs/rust
+crates/consensus*  @alpenlabs/chain-core
+crates/db          @alpenlabs/database
+crates/eectl       @alpenlabs/evm
+crates/evm*        @alpenlabs/evm
+crates/primitives  @alpenlabs/crypto @alpenlabs/rust
+crates/proof*      @alpenlabs/prover
+crates/prover      @alpenlabs/prover
+crates/reth        @alpenlabs/evm
+crates/rocksdb*    @alpenlabs/database
+crates/rpc         @alpenlabs/chain-core @alpenlabs/btc
+crates/state       @alpenlabs/chain-core
+crates/status      @alpenlabs/chain-core
+crates/storage     @alpenlabs/chain-core
+crates/sync        @alpenlabs/chain-core
+crates/tasks       @alpenlabs/chain-core
+crates/test-utils  @alpenlabs/rust @alpenlabs/ci
+crates/util*       @alpenlabs/rust
+crates/tx-parser   @alpenlabs/rust @alpenlabs/btc
+crates/*jmt*       @alpenlabs/database
```

Comment:
This appears to have only a single match.

Reply:
I'm trying to future-proof it